### PR TITLE
Fixed mismatch of the code with information in the console

### PR DESCRIPTION
**Pull request by bohov**

### CODE REVIEW

File: site/guide/2-getting-started.md [hunk 1]
```diff
@@ -81,13 +81,13 @@ run`. You should see the following:
     => address: localhost
     => port: 8000
     => log: normal
-    => workers: [logical cores * 2]
+    => workers: 16
```
Comment (SergioBenitez) [inline site/guide/2-getting-started.md]: This (and the other one) should be `[logical cores * 2]`, to keep it generic.

Comment (SergioBenitez) [inline site/guide/2-getting-started.md]:
```suggestion
    => workers: [logical cores * 2]
```

File: site/guide/3-overview.md [hunk 2]
```diff
@@ -162,13 +162,13 @@ Running the application, the console shows:
     => address: localhost
     => port: 8000
     => log: normal
-    => workers: [logical cores * 2]
+    => workers: 16
```
Comment (SergioBenitez) [inline site/guide/3-overview.md]:
```suggestion
    => workers: [logical cores * 2]
```

File: site/guide/2-getting-started.md (hunk 1 above)
Comment (bohov) [reply site/guide/2-getting-started.md]: Suggestion was accepted.

Comment (bohov) [reply site/guide/2-getting-started.md]: Changes have been accepted. Thanks for the info!

**Comment by SergioBenitez** [comment]
Merged in 81954532491b265ae8dd82ac6c3cd01b77f77849. Thanks!

**Pull request closed without merging**